modules/addons/src/webvr/display.js

```diff
@@ -0,0 +1,33 @@
+export default class Display {
```

Comment:
It isn't necessary to create a class hierarchy at this point since there's only one subclass. I'd prefer to keep the architecture simple until we see a need to extend.